install-all.sh

```diff
@@ -1,3 +1,5 @@
 npm i --legacy-peer-deps \
-&& npm i --legacy-peer-deps --prefix packages/yoroi-extension
+&& npm i --legacy-peer-deps --prefix packages/yoroi-extension \
+&& npm i --prefix packages/e2e-tests chromedriver --chromedriver_version=LATEST \
+&& npm i --prefix packages/e2e-tests
```

Comment:
@Nebyt , why this needs to happen twice?

Reply:
My mistake, the line 4 should be deleted.

Reply:
Oh, no, wait. This change is specifically for the chromedriver to install the latest one.